src/pages/settings/project/ProjectSettings.jsx

```diff
@@ -61,6 +66,42 @@ const ProjectSettings = () => {
       .finally(() => dispatch(decrement("pendingRequests")));
   };
 
+  const handleAes = AES => {
+    dispatch(increment("pendingRequests"));
+    client.projects
+      .setProjectGlobalConfig(projectID, {
+        AES,
+        secret: secret,
```

Comment:
use global config object (obtained by object destructuring the selectedProject) instead of passing these variables everytime